flake.nix

```diff
@@ -0,0 +1,95 @@
+{
+  description =
+    "Decentralized search engine & automatized press reviews - firefox extension";
+
+  # Nixpkgs / NixOS version to use.
+  inputs.nixpkgs.url = "nixpkgs/21.05";
+
+  inputs.meta-press = {
+    url = "https://framagit.org/Siltaar/meta-press-ext";
+    flake = false;
+    submodules = true;
+    type = "git";
+  };
+
+  outputs = { self, nixpkgs, meta-press }:
+    let
+      version = "1.7.7";
+
+      # System types to support.
+      supportedSystems = [ "x86_64-linux" ];
```

Comment:
This should work on other platforms, right? Might be worth adding ARM in there just so we have the target.